README.md

```diff
@@ -1 +1,109 @@
-# persistentvolumeclaim-storageclass-policy
+# Restrict StorageClasses in PersistentVolumeClaims
+
+This Kubewarden policy is designed to enhance the security and manageability of
+Kubernetes clusters by preventing the use of certain storage classes within
+`PersistentVolumeClaim` (PVC) objects. The policy provides an option to
+configure a fallback storage class, offering a seamless alternative when a
+denied storage class is requested.
+
+## Policy Objective
+
+The main goal of this policy is to restrict the usage of specific storage
+classes in PersistentVolumeClaims. Optionally, the policy can mutate the PVC to
+utilize a designated fallback storage class if a restricted storage class is
+requested. This policy does not evaluate PVCs that do not specify a
+`storageClassName`, as Kubernetes defaults to the cluster's default storage
+class in such cases.
+
+## Configuration
+
+The policy is configurable to meet the needs of different Kubernetes
+environments. Below is the structure of the policy's configuration parameters:
+
+```yaml
+# List of storage classes that are not allowed
+deniedStorageClasses:
+- fast
+- nvme
+
+# Optional: Specifies the fallback storage class to use when a denied storage class is requested
+fallbackStorageClass: slow
+```
+
+## How It Works
+The policy operates by evaluating the `storageClassName` specified in
+`PersistentVolumeClaim` objects. If a PVC requests a storage class listed in
+`deniedStorageClasses`, the policy action will depend on the configuration:
+
+- Without a `fallbackStorageClass` specified, the PVC will be rejected.
+- With a `fallbackStorageClass` specified, the PVC will be mutated to use the
+  fallback storage class, allowing the request to proceed.
+
```

Comment:
Add a note explaining that the fallback storage class cannot be on the deny list